packages/dnb-eufemia/src/extensions/forms/Form/Handler/Handler.tsx

```diff
@@ -17,6 +17,14 @@ export type Props = FormElementProps & {
    * Will decouple the form element from rendering
    */
   decoupleForm?: boolean
+
+  /**
+   * A wrapper component to wrap the form element.
+   * You can provide an array of components to wrap the form element from right to left.
```

Comment:
```suggestion
   * You can provide an array of components to wrap the form element from left to right.
```

Isn't this correct? Wouldn't right to left result in the last item in the array being at the top of the hierarchy?
Based on the tests, it seems like it works opposite. First item in array at top.